action.yml

```diff
@@ -8,11 +8,11 @@ inputs:
     description: 'Store: VAR=VALUE,OTHER_VAR=VALUE | Retrieve: VAR,OTHER_VALUE'
     required: true
   delimiter:
-    description: 'Regex defining the delimiter'
+    description: 'Regex delimiter between each variable'
     required: false
-    default: "\\r?\\n"
+    default: "\\r?\\n" # Regex splitting line with a '\n' or '\r'. Notice, we have to escape these characters because they are special and needs to be considered like it.
```

Comment:
# Regex splitting line with a '\n' or '\r'. They also need to be escaped, as they are special chars.